### PR TITLE
allow array of cap names to be passed and checked with carrier

### PR DESCRIPTION
**Pull request by dougperez**
## Ticket

https://jira.bqsoft.com/browse/ATOM-8402

## What this does

Allows array if cap names to be passed in 1 call.

## Why we did this

Helpful when checking if device is wifi capable for multiple types.

### CODE REVIEW

File: lib/conduit/sprint/actions/query_device_capabilities.rb [hunk 1]
```diff
@@ -5,7 +5,31 @@ class QueryDeviceCapabilities < Conduit::Driver::Sprint::Base
     wsdl_service        "WholesaleQueryDeviceInfoService/v1"
     xsd                 "wholesaleQueryDeviceCapabilities/v1/wholesaleQueryDeviceCapabilitiesV1.xsd"
     operation           :wholesale_query_device_capabilities_v1
-    required_attributes :meid, :capability_type_name
+    required_attributes :meid
+    optional_attributes :capability_type_name, :capability_type_names
```
Comment (pedromartinez) [inline lib/conduit/sprint/actions/query_device_capabilities.rb]: Let's always make them pass an array and make it required

File: lib/conduit/sprint/actions/query_device_capabilities.rb [hunk 2]
```diff
@@ -5,7 +5,31 @@ class QueryDeviceCapabilities < Conduit::Driver::Sprint::Base
     wsdl_service        "WholesaleQueryDeviceInfoService/v1"
     xsd                 "wholesaleQueryDeviceCapabilities/v1/wholesaleQueryDeviceCapabilitiesV1.xsd"
     operation           :wholesale_query_device_capabilities_v1
-    required_attributes :meid, :capability_type_name
+    required_attributes :meid, :capability_type_names
+    optional_attributes :capability_type_name
```
Comment (pedromartinez) [inline lib/conduit/sprint/actions/query_device_capabilities.rb]: Why do we still need this optional attribute?

File: lib/conduit/sprint/actions/query_device_capabilities.rb [hunk 3]
```diff
@@ -5,7 +5,33 @@ class QueryDeviceCapabilities < Conduit::Driver::Sprint::Base
     wsdl_service        "WholesaleQueryDeviceInfoService/v1"
     xsd                 "wholesaleQueryDeviceCapabilities/v1/wholesaleQueryDeviceCapabilitiesV1.xsd"
     operation           :wholesale_query_device_capabilities_v1
-    required_attributes :meid, :capability_type_name
+    required_attributes :meid, :capability_type_names
+
+    def attributes_with_values
```
Comment (pedromartinez) [inline lib/conduit/sprint/actions/query_device_capabilities.rb]: Can we simplify all those by only supporting arrays?

Comment (dougperez) [reply lib/conduit/sprint/actions/query_device_capabilities.rb]: Updated

**Pull request merged**